src/transformers/generation/configuration_utils.py

```diff
@@ -256,25 +255,21 @@ class GenerationConfig(PushToHubMixin):
             `batch_size`. Indicating a list enables different start ids for each element in the batch
             (e.g. multilingual models with different target languages in one batch)
 
-
-        > Generation parameters exclusive to [assistant generation](https://arxiv.org/abs/2211.17192)
+        > Generation parameters exclusive to assistant generation
```

Comment:
Links do not work within the doc section header:
<img width="492" alt="Screenshot 2024-05-02 at 12 24 11" src="https://github.com/huggingface/transformers/assets/12240844/315555c5-30a2-455f-bc87-c5023f4743cd">